.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (Problem/Solution)** in #expensify-open-source
+- [ ] Paste Proposal in the space above with a link to the Slack thread
+- [ ] Email `strategy@expensify.com` and paste in the Proposal
+- [ ] Fill out **High-level overview of the problem**, **Timeline**, and **Terminology** sections of the Design Doc
+- [ ] Email `strategy@expensify.com` (same email chain as before) with link to Design Doc
+- [ ] Host a _pre-design_ meeting in `#expensify-open-source` to discuss any necessary details in public before filling out the **High-level of proposed solution** section
+- [ ] Fill out the **High-level of proposed solution** section
+- [ ] Email `stategy@expensify.com` again with links to the doc and pre-design conversation in Slack
+- [ ] Add `DesignDocReview` label to get the **High-level of proposed solution** section reviewed
+- [ ] Respond to any questions or concerns and bring up blockers in Slack to get consensus if necessary
+- [ ] Confirm that the doc has the minimum necessary number of reviews before proceeding
+- [ ] Host another pre-design meeting in `#expensify-open-source` to ask for engineering feedback on the technical solution.
+- [ ] Fill out the **Detailed implementation of the solution** and related sections.
+- [ ] Re-add the `DesignDocReview`
+- [ ] Respond to any questions or concerns and bring up blockers in Slack to get consensus if necessary
+- [ ] Confirm that the doc has the minimum necessary number of reviews before proceeding
+- [ ] Email `strategy@expensify.com` one last time to let them know the Design Doc is moving into the implementation phase
+- [ ] Implement the changes
```

Comment:
We should add another checkbox for the post-mortem at the bottom of the doc

Reply:
Great idea!